src/osd/OpRequest.cc

```diff
@@ -93,6 +93,13 @@ void OpRequest::_dump_op_descriptor_unlocked(ostream& stream) const
   get_req()->print(stream);
 }
 
+uint64_t OpRequest::get_poolid() const
```

Comment:
Not sure this should be an accessor of OpRequest, as it is only relevant to FastDispatch messages.
Why not cast and access in the client code?